Integrate outstanding branches (#56)
WindhoverLabs/airliner: docs/glossary.rst

Glossary
====================================

.. glossary::
  
  AshaiKASEI AK8963

>     The AshaiKASEI AK8963 flight software application uses a 3-axis electronic compass to navigate. The app utilizes GPS to determine where the drone is at.

Reviewer: ```suggestion
    The AshaiKASEI AK8963 flight software application publishes samples from the AK8963 3-axis electronic compass.
```

  HK
    TODO: Add defintion of the Housekeeping (HK) Application

  HMC5883
    TODO: Add defintion of the Honeywell HMC5883 (HMC5883) Application

  HS
    TODO: Add defintion of the Health Services (HS) Application

  LC
    TODO: Add defintion of the Limit Checker (LC) Application

  LD
    TODO: Add defintion of the Landing Detector (LD) Application

  LGC
    TODO: Add defintion of the Landing Gear Control (LGC) Application

  MAC
    TODO: Add defintion of the Multicopter Attitude Control (MAC) Application

  MAVLINK
    TODO: Add defintion of the Mavlink Application

  MD
    TODO: Add defintion of the Memory Dwell (MD) Application

  MM
    TODO: Add defintion of the Memory Manager (MM) Application

  MPC
    TODO: Add defintion of the Multicopter Position Control (MPC) Application

  MPU6050
    TODO: Add defintion of the Invensense MPU6050 (MPU6050) Application 
    
  MPU9250
    TODO: Add definition of the Invensense MPU9250 (MPU9250) Application
  
  MS5607
    TODO: Add defintion of the TE Connectivity MEAS MS5607 (MS5607) Application

  MS5611 
    TODO: Add defintion of the TE Connectivity MEAS MS5611 (MS5611) Application 
  
  NAV

>     The Navigator Application (NAV) is responsible for safely navigating or moving the vehicle from one location to another.  

Reviewer: ```suggestion
TODO: Add definition for the Navigator (NAV) application.
```

  
  PBLIB
    TODO: Add defintion of the Proto Buffer Library (PBLIB) Application
  
  PE

>     The Position Estimator (PE) flight software application sends position data from the remote vehicle to the ground.

Reviewer: ```suggestion
    TODO: Add definition for the Position Estimator (PE) application.
```
